README.md

```diff
@@ -33,20 +33,30 @@ Swagger UI Version | Release Date | OpenAPI Spec compatibility | Notes
 
 ### How to run
 
-##### Easy start!  Docker
+##### Docker usage
 You can pull a pre-built docker image of the swagger-ui directly from Dockerhub:
 
 ```
 docker pull swaggerapi/swagger-ui
 docker run -p 80:8080 swaggerapi/swagger-ui
 ```
 
-Will start nginx with swagger-ui on port 80.
+This will start nginx with swagger-ui on port 80.
 
-Or you can provide your own swagger.json on your host
+###### Volumes
+When using Docker volumes, your able to sync your Swagger specification with the Docker container:
 
 ```
-docker run -p 80:8080 -e SWAGGER_JSON=/foo/swagger.json -v /bar:/foo swaggerapi/swagger-ui
+docker run -p 80:8080 -e "SWAGGER_JSON=swagger.json" -v /MyProject/spec/:/usr/share/nginx/html/spec/ swaggerapi/swagger-ui
```

Comment:
mounting the whole `/usr/share/.../spec` is long, And given the `docker run -p 80:8080 -e "SWAGGER_JSON=/spec/swagger.json" -v /MyProject/spec/:/spec/ swaggerapi/swagger-ui` example below. Which only mount `/spec`. I feel we should combine these two.

My suggestion is we add another nginx root. That will look inside `/spec` and default to `/spec/swagger.{yaml|json}`

Something as simple as 
```nginx
location /spec {
    index swagger.yaml swagger.json; # The index file may/may-not work. And is only a nice to have.
    root /spec
}
```

This means we just need to update the URL inside the index.html file to point to `/spec` or `/spec/$SWAGGER_JSON`. And we don't need to copy files over to /usr/share/nginx/...

So the examples become...
```sh
docker run -p 80:8080 -v /MyProject/spec:/spec swaggerapi/swagger-ui # Will use /MyProject/spec/swagger.json file ( $refs will be supported too, since this is the whole folder )


docker run -p 80:8080 -v -e "SWAGGER_JSON=petstore.yaml" /MyProject/spec:/spec swaggerapi/swagger-ui # Will use /MyProject/spec/petstore.yaml
```
The idea is only have one volume command. Less moving parts.

@guidsen let me know your thoughts on this. Or if anything I said was vague. And thank you for your efforts!